vnext/src/Libraries/AppTheme/AppThemeTypes.d.ts

```diff
@@ -22,3 +24,20 @@ export interface IHighContrastChangedEvent {
   isHighContrast: boolean;
   highContrastColors: IHighContrastColors;
 }
+
+export type AppTheme = {
+  addListener(
+    eventName: 'highContrastChanged',
+    listener: (nativeEvent: IHighContrastChangedEvent) => void,
+  ): EmitterSubscription;
+  removeListener(
```

Comment:
I think this form of emitter is now deprecated. Could you add a TS annotation pointing folks to the Emitter Subscription?